manifests/params.pp

```diff
@@ -272,9 +272,18 @@
   # - puppetdb for puppetdb
   $server_storeconfigs_backend = undef
 
-  $server_ssl_dir  = $ssldir
-  $server_package     = undef
-  $server_version     = undef
+  if ($::osfamily =~ /(FreeBSD|DragonFly)/) {
```

Comment:
I'd leave this `if` off. It doesn't hurt to always define `$puppet_major`.

Reply:
done.